### DISCUSSION TITLE
Fitting with xerrors

### DISCUSSION THREAD

**Discussion by Giorgio235** [scikit-hep/iminuit#717] [Q&A] [answered]
Hi,
I am a bit on a time-crunch so I am sorry if you already tackled this issue somewhere else.
I was wondering what is the best way to fit data points with errors on both x and y coordinates with a custom function.
thank you,
Giorgio

**Comment by HDembinski** [comment, marked as answer]
Here is an example. https://iminuit.readthedocs.io/en/stable/notebooks/automatic_differentiation.html#Fit-data-points-with-uncertainties-in-x-and-y
